manifests/0000_31_control-plane-machine-set-operator_01_rbac.yaml

```
@@ -0,0 +1,130 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: control-plane-machine-set-operator
+  namespace: openshift-machine-api
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: control-plane-machine-set-operator
+  namespace: openshift-machine-api
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+rules:
+  - apiGroups:
+      - machine.openshift.io
+    resources:
+      - machines
+    verbs:
+      - list
+      - watch
```

For Machines, we need the full suite as we will be responsible for creating, updating and deleting the machines
```suggestion
      - get
      - list
      - watch
      - create
      - update
      - patch
      - delete
```